<?xml version="1.0" ?>
<sdf version="1.4">
  <world name="default">
    <gui><camera name="user_camera"><pose>0 0 9.5 0 1.5 0</pose></camera></gui>

    <include><uri>model://sun</uri></include>
    <include><uri>model://ground_plane</uri></include>

    <model name="straightroad">
      <static>true</static>
      <link name="link">
	<visual name="bar1">
	  <pose>0 0 0 0 0 0</pose>
	  <geometry>
	    <box><size>5 1.2 .001</size></box>
	  </geometry>
	  <material>
	    <ambient>0.2 0.2 0.2 1</ambient>
	    <diffuse>0.2 0.2 0.2 1</diffuse>
	    <specular>0.2 0.2 0.2 1</specular>
	    <emissive>0.2 0.2 0.2 1</emissive>
	  </material>
	</visual>
      </link>
      <link name="link2">
	<visual name="bar2">
	  <pose>0 0 0 0 0 0</pose>
	  <geometry>
	    <box><size>5 0.12 .0015</size></box>
	  </geometry>
	  <material>
	    <ambient>0.5 0.5 0 1</ambient>
	    <diffuse>0.5 0.5 0 1</diffuse>
	    <specular>0.5 0.5 0 1</specular>
	    <emissive>0.5 0.5 0 1</emissive>
	  </material>
	</visual>
      </link>
    </model>
  </world>
</sdf>
